modules/nw-metalLB-basic-upgrade-operator.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * networking/metallb/metallb-upgrading-operator.adoc
+
+:_content-type: PROCEDURE
+
+[id="upgrading-metallb-operator-to-minor-version_{context}"]
+= Upgrading the MetalLB Operator
+
+Manually upgrade the MetalLB Operator to the next minor version by using the {product-title} web console.
+
+.Prerequisites
+
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Access the web console and navigate to *Operators* -> *Installed Operators*.
```

Comment:
For example this section is not required if we are describing the upgrade process from 4.10, because we are asking to remove the operator anyway

Reply:
Do we need this? If we are upgrading from < 4.11, we'll have to delete so there's no point in changing the approval strategy.